medicines/web/components/search-results/index.tsx

```diff
@@ -130,6 +131,23 @@ const searchResultsTitle = (
     : `Showing results for ${showingResultsForTerm}`;
 };
 
+interface ISearchNumberingInformation {
```

Comment:
`IPaginationMetadata`?

Reply:
that seems less specific to me, although I agree `Pagination` is better to have in the name than `SearchNumbering` which isn't as clear, I'll retool this name

Reply:
on reflect, this is being used to give users a message about what search results numbers they're seeing, and I'd like to keep the name as-is for now to describe that purpose

Reply:
`ISearchResultsMetadata`? :art: